src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -239,7 +241,7 @@ private void GroupFixes(Workspace workspace, IEnumerable<CodeFixCollection> fixC
                             if (fix.Action is SuppressionCodeAction)
                             {
                                 var suggestedAction = new SuppressionSuggestedAction(workspace, _subjectBuffer, _owner._editHandler,
-                                    fix, fixCollection.Provider);
+                                    fix, fixCollection.Provider, getFixAllSuggestedActionSet);
```

Comment:
Why do we need to defer the calculation of the SuggestedActionSet in the suppression case? Why not calculate the SuggestedActionSet and pass it to SuppressionCodeAction as we were doing (and are still doing) for CodeFixSuggestedAction above?